registry/handlers/referrers.go

```diff
@@ -195,6 +231,19 @@ func readlink(ctx context.Context, path string, stDriver driver.StorageDriver) (
 	return digest.Parse(string(content))
 }
 
+func generateLinkHeader(repoName, subjectDigest, artifactType string, pageSize int, p int) string {
+	linkURL := fmt.Sprintf("/v2/%s/referrers/%s", repoName, subjectDigest)
+	v := url.Values{}
+	v.Add("p", fmt.Sprint(p))
```

Comment:
nit: prefer [`strconv.Itoa`](https://pkg.go.dev/strconv#Itoa) to `fmt.Sprint` for performance.
```suggestion
	v.Add("p", strconv.Itoa(p))
```

Reply:
changed.